subsys/bluetooth/controller/ll_sw/lll.h

```diff
@@ -173,8 +173,14 @@ enum node_rx_type {
 	NODE_RX_TYPE_MESH_ADV_CPLT = 0x13,
 	NODE_RX_TYPE_MESH_REPORT = 0x14,
 #endif /* CONFIG_BT_HCI_MESH_EXT */
-};
 
+/* Following proprietary defines must be at end of enum range */
+#if defined(CONFIG_BT_PROPRIETARY_EXT)
+	NODE_RX_TYPE_PROPRIETARY_START = 0x15,
+	NODE_RX_TYPE_PROPRIETARY_END = NODE_RX_TYPE_PROPRIETARY_START + 0x5,
```

Comment:
0x5 should be a(nother) kconfig.

Reply:
I added a config (BT_CTLR_USER_EXT _RANGE) in subsys/bluetooth/controller/Kconfig